jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -111,29 +128,28 @@ docker_images := {image | walk(input, [path, value])  # walk the entire config t
                           path[_] == "docker"         # find any settings that match 'docker'
                           image := value[_].image}    # grab the images from that section
 
-```
+----
 
-### Enforcement
-{: #enforcement }
+===== Enforcement
 
 The policy service allows rules to be enforced at different levels.
 
-#TODO: add default level which is soft fail
+#TODO: add default level which is soft fail#
```

Comment:
@sagar-connect could you check if this "To do" has been addressed?

Reply:
fixed